web/client/themes/default/less/geo-processing-tools.less

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2023, GeoSolutions Sas.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// **************
+// Theme
+// **************
+#ms-components-theme(@theme-vars) {
+    // #GeoProcessingTools-root {
+
+    // }
+}
```

Comment:
I don't think this is needed. 
you can remove it.

Reply:
no we usually keep it empty

Reply:
maybe i can comment it eventually but not remove it

Reply:
Sorry, wher it is written that it have to be  keep it, even if empty.
I may have missed something.

Reply:
it's more of a convention

@allyoucanmap do you remember if it's written somewhere, training or docs?

you can find more examples
https://github.com/geosolutions-it/MapStore2/blob/3e1d020e5799de91d61a141e2bae6835456ac13b/web/client/themes/default/less/catalog.less#L9-L15

https://github.com/geosolutions-it/MapStore2/blob/3e1d020e5799de91d61a141e2bae6835456ac13b/web/client/themes/default/less/colorrangeselector.less#L9-L17

Reply:
@MV88 not sure if it was mentioned in docs, probably those two examples are a copy and paste action. We could remove them as suggested by @offtherailz